### PR TITLE
Add minutes for 24 Nov 2020 meeting

### PR DESCRIPTION
**Pull request by SidFaber**
Signed-off-by: Sid Faber <sid.faber@canonical.com>

### CODE REVIEW

File: meetings/2020_11_24/README.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+[Kyle Fazzari](https://github.com/kyrofa),
+[Marco Gutierrez](https://github.com/marcoag),
+[Mikael Arguedas](https://github.com/mikaelarguedas),
+[Ruffin White](https://github.com/ruffsl),
+[Sid Faber](https://github.com/sidfaber)
+
+
+## Administrivia
+
+Following a brief discussion, it was decided to move new meeting minutes to the [ros-security/github repo](https://github.com/ros-security/community). Existing meeting minutes in the [ROS wiki](http://wiki.ros.org/ROS2/WorkingGroups/Security) will not be ported.
```
Comment (mikaelarguedas) [inline meetings/2020_11_24/README.md]:
```suggestion
Following a brief discussion, it was decided to move new meeting minutes to the [`ros-security/community` Github repository](https://github.com/ros-security/community). Existing meeting minutes in the [ROS wiki](http://wiki.ros.org/ROS2/WorkingGroups/Security) will not be ported.
```

File: meetings/2020_11_24/README.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+- Design input for running ROS without a file system
+- Reducing permission file size / complexity
+- Fixing test failures
+
+## Open Discussion
+ROS launch status: the initial launch is working but does not include access control. The work is in progress, but stalled pending discussions on [launch_ros PR 180](https://github.com/ros2/launch_ros/pull/180). Some comments are suggesting a plugin solution, which would change future PRs.
+
+Marco suggested [the Robotics Middleware Framework (RMF)](https://github.com/osrf/rmf_demos) as a reference implementation for ROS security. This should be ready to run with ROS 2; they have already done some work with security as well.
+
+Marco also asked about revoking keys: there's a need to handle that within RMF should an individual robot in a fleet be physically compromised. @JaimeMartin provided [information on CRLs from eProsima](https://fast-dds.docs.eprosima.com/en/latest/fastdds/security/auth_plugin/auth_plugin.html#generating-the-certificate-revocation-list-crl).
```
Comment (mikaelarguedas) [inline meetings/2020_11_24/README.md]: We should use an homogeneous way to refer to individuals in the notes (Github handles, first name, initials...).

File: meetings/2020_11_24/README.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+## Administrivia
+
+Following a brief discussion, it was decided to move new meeting minutes to the [ros-security/github repo](https://github.com/ros-security/community). Existing meeting minutes in the [ROS wiki](http://wiki.ros.org/ROS2/WorkingGroups/Security) will not be ported.
+
+The [vulnerability remediation procedure PR](https://github.com/ros-security/community/pull/8) is still open for comments.
+
+
+## G-Turtle goals
+
+Five open items could become part of our G-Turtle deliverables:
```
Comment (mikaelarguedas) [inline meetings/2020_11_24/README.md:40]: One thing mentioned in the last meetings and that I'd like us to look into is the ability of having configurable security levels per topic, following the ones supported by DDS-Security: NONE SIGN, ENCRYPT
Currently SROS2 is all or nothing, either all topics are encrypted or no security feature is used at all. This doesn't scale well (or makes sense) for real systems. Tracking ticket and past work on the topic available at https://github.com/ros2/sros2/issues/130

File: meetings/2020_11_24/README.md (hunk 1 above)
Comment (mikaelarguedas) [inline meetings/2020_11_24/README.md]: Porting the past notes and replacing the content of the wiki page to point to this github repo would be preferred. although I understand it's cumbersome work

File: meetings/2020_11_24/README.md (hunk 2 above)
Comment (mikaelarguedas) [inline meetings/2020_11_24/README.md]:
```suggestion
Marco also asked about revoking keys: there's a need to handle that within RMF should an individual robot in a fleet be physically compromised. Jaime provided [information on CRLs from eProsima](https://fast-dds.docs.eprosima.com/en/latest/fastdds/security/auth_plugin/auth_plugin.html#generating-the-certificate-revocation-list-crl).
```

File: meetings/2020_11_24/README.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+This example will also be useful for us to find issues with the security implementation on a complex system to test: CPU / network utilization, what to sign, what to encrypt, overall impact to the system.
+This also becomes a proving ground for NoDL.
+
+Simulation may be challenging; a simulated implementation may not quite match the real world implementation. However, we should be able to spec the project in stages. Start simple and build upon the demo.
+
+### Enable DDS security without a file system
+
+The scope of this issue is much wider than just security. Success depends upon buy-in from both the micro-ROS community and from Open Robotics.
+
+The agrees to continue to move the discussion forward to flesh out a design, but not to perform any work on the code at this time.
```
Comment (mikaelarguedas) [inline meetings/2020_11_24/README.md]:
```suggestion
The WG agrees to continue to move the discussion forward to flesh out a design, but not to perform any work on the code at this time.
```

**Review by mikaelarguedas — COMMENTED** [review]
thanks @SidFaber for putting these together

File: meetings/2020_11_24/README.md (hunk 3 above)
Comment (SidFaber) [reply meetings/2020_11_24/README.md:40]: Added to the "reference implementation" goal in a6e2e0e927a898d850106dbe5e7725534954701b--as I understood the conversation, we need a good implementation with some complexity before we can begin configuring topic-level security.

File: meetings/2020_11_24/README.md (hunk 1 above)
Comment (SidFaber) [reply meetings/2020_11_24/README.md]: Updated in a6e2e0e927a898d850106dbe5e7725534954701b

File: meetings/2020_11_24/README.md (hunk 2 above)
Comment (SidFaber) [reply meetings/2020_11_24/README.md]: Agreed--I experimented with github handles, but they don't carry through to markdown. Plan on linking to github profiles in the "attendee" list, but then using first names throughout similar to the style used on the wiki site.

File: meetings/2020_11_24/README.md (hunk 1 above)
Comment (SidFaber) [reply meetings/2020_11_24/README.md]: I'll leave a few breadcrumbs to cross-link the wiki and the github repo, but I think the effort to port the WG content would be better spent on updating sros2 documentation.

File: meetings/2020_11_24/README.md (hunk 2 above)
Comment (SidFaber) [reply meetings/2020_11_24/README.md]: Updated in a6e2e0e927a898d850106dbe5e7725534954701b

File: meetings/2020_11_24/README.md (hunk 4 above)
Comment (SidFaber) [reply meetings/2020_11_24/README.md]: Updated in a6e2e0e927a898d850106dbe5e7725534954701b

**Comment by SidFaber** [comment]
Thanks for the thorough read-though, @mikaelarguedas !

**Review by kyrofa — COMMENTED** [review]
This looks good to me, but I'm curious why you made the meeting a directory with a README instead of just a dated md file. In case we have some other collatoral during a meeting? If so, can we at least rename README to something like `minutes.md`?

**Comment by SidFaber** [comment]
Directories so we can attache things (presentations?) to the meetings. README just to reduce clicks--the minutes render as soon as you land in the directory.

**Review by kyrofa — APPROVED** [review]
Oh fair point, okay.

**Pull request merged**